src/clj/clojurians_log/db/queries.clj

```diff
@@ -64,4 +64,14 @@
          [?user :user/slack-id ?id]
          [?user :user/name ?username]]
        db
+       names))
+
+(defn channel-names
+  [db names]
```

Comment:
The parameter `names` seems a bit misleading here. I think it'd be clearer if it's named something like `ids` or `channel-ids`.